README.md

```diff
@@ -27,21 +41,22 @@ The Extensions API lets you do more without leaving Tableau. Build Tableau exten
 
    **npm start**
 
-5. Launch Tableau and try the sample extensions in a dashboard. The samples are located in the `Samples` folder.
+5. Launch Tableau and try a dashboard sample extension in a dashboard, or a viz extension in a worksheet. The dashboard and viz extension samples are located in the `Samples` folder.
 
- >**Note** The local web server you start just serves to host the extension samples and extensions used in the tutorial, which have URLs similar to the following: `http://localhost:8765/Samples/DataSources/datasources.html` or `http://localhost:8765/Samples-Typescript/DataSources/datasources.html`
- >   This local web server is not intended to serve the Extensions API Help pages. 
+ >**Note** The local web server you start just serves to host the extension samples and dashboard extensions used in the tutorial. These extensions have URLs similar to the following: `http://localhost:8765/Samples/Dashboard/DataSources/datasources.html`.
```

Comment:
Not sure if there are plans to have a Viz extensions tutorial, but the local web server should be able to host the viz extensions tutorial if one ever gets added. So it might not be necessary to specify that it only the hosts dashboard extensions tutorial. I think it might be fine to change it to just say "... and extensions used in the tutorial."